### ISSUE TITLE
Unable to login to trello.com - v1.0.7.7

### ISSUE THREAD

**Issue by dperson** [andryou/scriptsafe#28]
The only item listed as blocked is `<NOSCRIPT>`, but the `log in` button is greyed out unless I disable ScriptSafe.

FYI, I manually told chrome (51.0.2704.79) to check for updated extensions.

Thanks for the great extension.

**Comment by dperson** [comment]
Just ran into another issue this time with `https://developer.servicenow.com/app.do` saying that something is blocking it's javascript. Nothing is listed as being blocked by ScriptSafe, but when I disable it, the site works...

**Comment by andryou** [comment]
Thanks for the report.

Have you allowed trello.com? (added it to your whitelist) Or do you have the "Respect Same-Domain" enabled?

If the latter, it's a  known issue that is fixed in #25 and will be pushed to your browser once it's approved in the web store.

**Comment by dperson** [comment]
I do believe that is the issue, I'll close this one.

Thanks for the quick reply.
